core/src/test/java/mediumtest/AuthenticationMediumTests.java

```diff
@@ -49,7 +49,7 @@
 class AuthenticationMediumTests {
 
   @RegisterExtension
-  WireMockExtension sonarqubeMock = WireMockExtension.newInstance()
+  static WireMockExtension sonarqubeMock = WireMockExtension.newInstance()
```

Comment:
Regarding such changes, it should probably be done also in `SslMediumTests` and `ProxyMediumTests`

Reply:
Indeed. FYI I am fixing it on-the-fly since this is a very minor performance detail (the local HTTP server will be restarted between each tests)